### DISCUSSION TITLE
How to show `pyvis` network graph?

### DISCUSSION THREAD

**Discussion by rickhg12hs** [microsoft/vscode-jupyter#12381] [Questions and Answers]
I have some large networks that I'd like to show within vscode with `pyvis`.  Unfortunately, I can't even get the `pyvis` tutorial to work.

Here's the simple `pyvis` `jupyter` example from their [tutorial](https://pyvis.readthedocs.io/en/latest/tutorial.html#using-pyvis-within-jupyter-notebook).  When I try it vscode, nothing is displayed.  How can `pyvis` network graphs be shown in `vscode-jupyter`?

![pyvis jupyter tutorial](https://pyvis.readthedocs.io/en/latest/_images/jup.png)

**Comment by DonJayamanne** [comment]
This is an issue in the Jupyter extension, please check here for udpates https://github.com/microsoft/vscode-jupyter/issues/12689
